buildconfig/manylinux-build/docker_base/glib/glib.sha512

```diff
@@ -1 +1 @@
-280a46c2af13283a08c15ff0b4f5492659c2884521930600ad45310ed181c44a878ad8f9b36bae68ed6e7d92db6f1630f7bf015148c513dc317d25807f13abb0  glib-2.56.4.tar.xz
+1514d62aeb4c4a1a1048ae0f84f7db7f0dbf355772b2dadf6a34ec547045b163a5e28331b096e7616fe3c9c19bed98025a0202b05073f5d7ee901d0efaffe143  glib-2.80.0.tar.xz
```

Comment:
```suggestion
1514d62aeb4c4a1a1048ae0f84f7db7f0dbf355772b2dadf6a34ec547045b163a5e28331b096e7616fe3c9c19bed98025a0202b05073f5d7ee901d0efaffe143  glib-2.80.0.tar.xz

```

Reply:
GitHub puts a 🚫 on files that do not end with a single blank line.

Reply:
Thanks. That's fixed now.